src/problem_bank_scripts/tui/tui.py

```diff
@@ -99,33 +112,37 @@ def other_asks(part: dict, solution: str, use_gpt: bool, exercise: dict | None =
         case "multiple-choice" | "dropdown" | "checkbox":
             options: list[str] = []
             # answer = questionary.text("Solution").ask()
-            num_options = ask_int("Number of options (excluding solution)", default=3)
+            num_options = ask_int("Number of options", default=4)
             for i in range(num_options):
                 options.append(
                     questionary.text(f"Option {i+1}. Press enter to generate with GPT.").ask()
                 )
             info["choices"] = generate_given_choices(options, solution, question, use_gpt)
+            if key == "dropdown":
+                key = "multiple-choice"
+                part["type"] = "multiple-choice"
+                info["display"] = "dropdown"
```

Comment:
couldn't some of this be set in the QUESTION_TYPES dict?
i.e:
```py
...
"dropdown": {"type": "multiple-choice", "display": "dropdown"},
...
```
also, I don't understand the `key = "multiple-choice"` line, that doesn't seem to be used anywhere after the reassignment?